### ISSUE TITLE
I had to change the device id and vendor

### ISSUE THREAD

**Issue by monkiki** [Gibtnix/MSIKLM#13]
Hi,

I had to change the device id and vendor to get my keyboard detected:
```
dev = hid_open(0x1038, 0x1122, 0);
```
But I can't change the color :cry: 

My laptop is a GE73 7RD Raider.

**Comment by Gibtnix** [comment]
Hard to say if the commands are actually the same if already the IDs are different... I've also prepared a test version for [this issue](https://github.com/Gibtnix/MSIKLM/issues/12) where a slightly different command is used - feel free to give this version a try. If it works, I can prepare a different version for you. If not I need to know how the command syntax for the respective keyboard to get it working...

**Comment by monkiki** [comment]
I tried the other one, but the result is the same. It's quite odd that these laptops have different keyboard controllers.

**Comment by Gibtnix** [comment]
Indeed... but I don't know how to fix this issue then. If you find out something new here I will integrate it - sorry :-/

**Comment by monkiki** [comment]
Where did you get the information to create the current version? I mean, how to manage the colours and other features of the keyboard?

**Comment by Gibtnix** [comment]
Actually the first version only contained a basic configuration (still contained in msiklm.c, but currently commented out, cf. function 'set_color()'). I found the required commands somewhere and created the program using them, but they later have replaced to also allow full RGB color setting, see the respective [pull request](https://github.com/Gibtnix/MSIKLM/pull/4). I later unified the whole program to this command structure.

I think there are only two ways to find out how the communication with the keyboard has to be performed: One is to directly obtain some kind of interface description from MSI, SteelSeries or somewhere else and integrate it, the other one is to reverse-engineer the interface by logging the USB traffic on windows... feel free to find it out, if I knew how to do, I would update the program accordingly.

**Comment by monkiki** [comment]
Interesting. I will look for information and let's see if I can contribute with a fix.

Best regards.

**Comment by Gibtnix** [comment]
Perfect, hope you find a solution ;-)

**Comment by eduardoleal** [comment]
Hello, @monkiki. Did you get any luck on that? Tks

**Comment by monkiki** [comment]
I didn't find any useful information about the SteelSeries keyboard :(
